code/datums/uplink_item.dm

```diff
@@ -1268,6 +1268,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/chameleon
 	cost = 7
 
+/datum/uplink_item/stealthy_tools/chameleon_counter
+	name = "Chameleon Counterfeiter"
+	desc = "This device disguises himself as any object scanned by it. It's not a perfect replica and can be noticed when examined by someone."
```

Comment:
```suggestion
	desc = "This device disguises the user as any object scanned by it. The disguise is not a perfect replica and can be noticed when examined by an observer."
```

Reply:
"This device disguises himself as any object scanned by it. The disguise is not a perfect replica and can be noticed when examined by an observer."
This description works? The device is the one being altered not the user.

Reply:
👍